### ISSUE TITLE
0 Call to a member function enqueueMessage() on null 

### ISSUE THREAD

**Issue by ZerooCool** [codeling/bfstop#217]
Lire logs depuis administrator/composant cliquer sur Log :
```
Une erreur est survenue
    0 Call to a member function enqueueMessage() on null 
```

Messages erreur Apache2 :
```
# PHP Warning:  Undefined variable $msg         in .../administrator/components/com_bfstop/controllers/allow.php on line 17', referer: https://joomla.amis-sh.fr/
# PHP Warning:  Undefined variable $application in .../administrator/components/com_bfstop/models/log.php on line 49', referer: https://joomla.amis-sh.fr/
```

Un problème de referer sur ma configuration des headers, il manque le chemin /administrator dans l'URL referer ?
Ou un autre problème d'après vous ?

**Comment by codeling** [comment]
The immediate error you see comes from a problem in the code handling a specific case - when log lines do not have the expected format. This will be fixed in a future release. Check the content of the log file ( logfolder-configured-in-joomla/plg_system_bfstop.log.php for the actual problem - the content of the log file.  From the problematic code being triggered I can say that there's at least one line in there which does not have the 3 expected columns

**Comment by codeling** [comment]
Will be fixed in v1.5.3.
